src/vstart.sh

```diff
@@ -60,10 +60,10 @@ export DYLD_LIBRARY_PATH=$CEPH_LIB:$DYLD_LIBRARY_PATH
 [ -z "$CEPH_NUM_MON" ] && CEPH_NUM_MON=3
 [ -z "$CEPH_NUM_OSD" ] && CEPH_NUM_OSD=3
 [ -z "$CEPH_NUM_MDS" ] && CEPH_NUM_MDS=3
-[ -z "$CEPH_NUM_MGR" ] && CEPH_NUM_MGR=0
+[ -z "$CEPH_NUM_MGR" ] && CEPH_NUM_MGR=1
```

Comment:
ceph-mgr is not included in the "tests" target, see https://github.com/ceph/ceph/blob/master/src/test/CMakeLists.txt#L503. if we want to start mgr by default, we need to include it in "tests".

```
Starting mgr.x
/home/jenkins-build/build/workspace/ceph-pull-requests/build/bin/ceph-mgr -i x
2017-02-02 15:26:32.875139 7f8a4081a300 -1 WARNING: all dangerous and experimental features are enabled.
2017-02-02 15:26:32.875297 7f8a4081a300 -1 WARNING: all dangerous and experimental features are enabled.
2017-02-02 15:26:32.969390 7f980ad29700 -1 WARNING: all dangerous and experimental features are enabled.
2017-02-02 15:26:32.992809 7f980ad29700 -1 WARNING: all dangerous and experimental features are enabled.
/home/jenkins-build/build/workspace/ceph-pull-requests/src/vstart.sh: 329: /home/jenkins-build/build/workspace/ceph-pull-requests/src/vstart.sh: /home/jenkins-build/build/workspace/ceph-pull-requests/build/bin/ceph-mgr: not found

        Start  11: test_pidfile.sh
  4/160 Test   #5: cephtool-test-mon.sh ....................***Failed   14.31 sec
```

see https://jenkins.ceph.com/job/ceph-pull-requests/17881/consoleFull#12243436770189bba6-5c6e-4283-b974-1eceae3f5f28